module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,12 @@
+#include "vowel.hpp"
+
+#include <regex>
+
+void removeVowels(std::vector<std::string>& words) {
+    std::regex vowels("[AEIOUYaeiouy]");
```

Comment:
You can make it constexpr and put it outside of function.

Reply:
Could be, but it would exist outside the function. Would it be an advantage?

Reply:
It's not neccesary here,but I would consider puting it in the header since it's important const,which you would like to be easily accesible if you would work in the larger project.

Reply:
Yes, that's for sure. Thank you for your apt comment!